packages/gestalt/src/Checkbox/InternalCheckbox.js

```diff
@@ -0,0 +1,260 @@
+// @flow strict
+import {
+  forwardRef,
+  type Node,
+  type AbstractComponent,
+  useImperativeHandle,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import classnames from 'classnames';
+import Box from '../Box.js';
+import colors from '../Colors.css';
+import focusStyles from '../Focus.css';
+import Icon from '../Icon.js';
+import Label from '../Label.js';
+import FormErrorMessage from '../shared/FormErrorMessage.js';
+import FormHelperText from '../shared/FormHelperText.js';
+import Text from '../Text.js';
+import useFocusVisible from '../useFocusVisible.js';
+import styles from './InternalCheckbox.css';
+
+type Props = {|
+  /**
+   * Indicates whether or not Checkbox is checked. See the [state variant](https://gestalt.pinterest.systems/web/checkbox#State) to learn more.
```

Comment:
These comments are no longer necessary now that this is an internal component, as they will never be surfaced to users. They can all be removed, unless there are any that are particularly useful to Future Us.